index.js

```diff
@@ -34,7 +34,7 @@ module.exports = function(options = {}) {
     return config.isListForm ? [] : {};
   }
 
-  const results = traverse(config);
+  const results = module.exports._traverse(config);
```

Comment:
Not super happy with this approach, but I don't know how to achieve the same with another package and rewire is buggy and sort of unmaintained.

If someone knows another package to replace rewire and achieve the same result without exporting stuff just for testing, let me know.